.github/workflows/test.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Environment setup
-        uses: defenseunicorns/uds-common/.github/actions/setup@5e4414dc25302739063bb58aa96b8afef5be9851 # v0.10.3
+        uses: defenseunicorns/uds-common/.github/actions/setup@0a1622c9ff2298722b9e47563236ac56d1a93a56 # v0.3.11
 
       - name: Test building the docker image
         run: uds run dev-build
```

Comment:
One last test - can you add `--no-progress` at this layer? I did this recently on a separate PR and had to add at this layer (https://github.com/defenseunicorns/uds-rke2-image-builder/pull/48)